anime_downloader/commands/watch.py

```diff
@@ -38,9 +38,14 @@
     help='The anime provider (website) for search.',
     type=click.Choice(sitenames)
 )
+@click.option(
+    '--mal_import',
+    help='Import xml file from MAL export.',
+    type=bool,
+    is_flag=True)
```

Comment:
Don't use is_flag=True.
With false you can get an argument from the command line without hacks.

Reply:
Will do, can you explain what the variable I would have to call would be then, just argv[0]?

Reply:
if you remove is_flag=True the argument after it becomes mal_import
![image](https://user-images.githubusercontent.com/46196380/89357757-bc099c00-d6c1-11ea-8378-94def6abf32c.png)

Reply:
Nice, good change

Reply:
On implementation, I lose the feature to prompt the user to add the path if it's empty. So I'm not making this change.